.github/workflows/schedule_tests.yml

```
@@ -195,3 +195,43 @@ jobs:
         working-directory: ./tests/
         run: |
           python -Wall runtests.py --verbosity 2 --noinput --selenium=chrome --headless --settings=test_postgres --parallel 2
+
+  supported-postgresql:
```

```suggestion
  postgresql:
```
Maybe? As we don't support 17 yet

---

Fixed! I also ensure that we test with and without server binding to fully exercise the error we were getting in the referenced ticket.